associations.rb

```diff
@@ -13,26 +13,27 @@ def migrate
 
 def generate_migrations
   ActiveRecord::Migration.create_table :hotels do |t|
-    #insert our columns here
-
+    t.string :name
+    t.integer :room_count
     t.timestamps null: false
   end
 
   ActiveRecord::Migration.create_table :rooms do |t|
-    #insert our columns here
-
+    t.integer :rate
```

Comment:
I prefer decimal too.

Some folks like to use an integer number of cents. Which is fine if you're using a tool to handle mapping the data between the db and your app's "money" object.